get-started/versioning/04-saving-and-retrieving-data.md

```diff
@@ -1,17 +1,17 @@
-In order to see the status of tracked data and model files and if they are
-stored in remotes, we can use 
+# Saving and Retrieving Data
 
-`dvc status --cloud`{{execute}}
+The command `dvc status` can compare the data on the workspace with the data saved
+on the cache. 
 
-to compare files between the data stored on cache and the default remote.
+`dvc status --cloud`{{execute}}
 
 Without the `-c, --cloud` option 
 
 `dvc status`{{execute}}
 
-checks whether workspace files are up to date with _local cache._
+checks whether workspace files are up to date with the _local cache._
```

Comment:
This order is still confusing IMO. Why not use plain `status` first and `-c` after?

Reply:
Or just mention plain `status` first (without executable command) and go on to using `status -c` which is what matters at this point. Plain `status` is used later.